tests/copytests/cases/test044.sh

```
@@ -0,0 +1,15 @@
+# copy a doubly nested directory containing a file to a non-existant directory with the -r flag
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)